src/page-files/setup.node.ts

```diff
@@ -26,9 +26,9 @@ async function setPageFiles(): Promise<unknown> {
     let moduleExports: any
     try {
       moduleExports = await ssrEnv.viteDevServer.ssrLoadModule(modulePath)
-    } catch (err) {
-      ssrEnv.viteDevServer.ssrFixStacktrace(err)
-      throw err
+    } catch {
+      // Vite logs the error, so just return empty array.
+      return []
```

Comment:
How about

```js
err._alreadyLoggedByVite = true
throw err
```

Reply:
Sounds good. We'll need a try-catch around this call too then:
https://github.com/brillout/vite-plugin-ssr/blob/16d6b56bf2c1860c62c36888bb769f1cc57a2ec2/src/renderPage.node.ts#L361